web-ide/ide-server/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "daml.telemetry": "Disable"
+}
```

Comment:
Can we get a trailing newline on this file?

Reply:
Did you not also want to reroute some shortcuts?

Reply:
thank you for reminding me!